R/02-filter.R

```diff
@@ -92,6 +100,8 @@ qc_filter_plasmid <- function(gimap_dataset, cutoff = NULL, filter_plasmid_targe
   
   if (is.null(filter_plasmid_target_col)) {filter_plasmid_target_col <- c(1)}
   
+  #@Howard please help: should set up a check that if filter_plasmid_target_col is not null, it's an R vector (? -- the c() thing) of integers greater than or equal to 1 and less than or equal to number of possible columns in data
```

Comment:
Can use similar logic to above.

Reply:
I'll push a change with similar logic and then merge if that's ok

Reply:
Sounds good to me!